apps/requestron/lib/jwt.js

```diff
@@ -0,0 +1,17 @@
+const { createAppAuth } = require('@octokit/auth-app');
```

Comment:
thought for next time: would consider using octokit/app directly for a singleton to avoid the extra auth token step in each call https://www.npmjs.com/package/@octokit/app